docs/concepts/events.md

```diff
@@ -55,11 +55,11 @@ An example event would look like:
 
 The first `A` means the event is originating from a contract, which will always be the case for user-defined events. The contract address as the name implies is the location of a contract deployed on the Flow network. Next is the name of the contracted event originates from, and last is the event type defined in the contract.
 
-There are an unlimited amount of events that can be defined on Flow, but you should know about the most common ones. 
+There is an unlimited amount of events that can be defined on Flow, but you should know about the most common ones. 
 
-### Flow Token Events
+### FLOW Token Events
 
-The Flow Token contract is the contract that issues a core Flow token. As with any contract, it can emit events when interacted with. When we transfer the Flow token, events are emitted. You can find a lot of details on the events emitted in the [Flow Token documentation](../concepts/core-contracts/03-flow-token.md). 
+The FLOW Token contract is the contract that issues a core FLOW token. As with any contract, it can emit events when interacted with. When we transfer the FLOW token, events are emitted. You can find a lot of details on the events emitted in the [FLOW Token documentation](../concepts/core-contracts/03-flow-token.md). 
```

Comment:
Should it be uppercase? not sure, it's not an abbreviation the same way as Dollar and USD

Reply:
market names of tokens are usually in capital case, like BTC, ETH, SHIB, MATIC:
https://www.coinbase.com/price/flow